sklearn/discriminant_analysis.py

```diff
@@ -255,6 +256,36 @@ def __init__(self, solver='svd', shrinkage=None, priors=None,
         self.store_covariance = store_covariance  # used only in svd solver
         self.tol = tol  # used only in svd solver
 
+    def __get__(self, obj, objtype):
```

Comment:
I don't think you mean this. defining `__get__` turns the class into a descriptor...